backend/src/core/features/game-identifier/game.identifier.js

```diff
@@ -34,102 +34,63 @@ export class GameIdentifier {
     this.#options = options;
   }
 
-  tryViaSteamWeb = async () => {
-    this.#logger.debugc("identifying games via steam web");
+  tryIfGameViaSource = async (source) => {
+    this.#logger.debugc(`identifying games via ${source}`);
```

Comment:
steamAppsAggregate data model

Put game.identifier code from line 45-60 there as method

Pass in logger into steam apps repo, and then to new aggregate data model

Use steamappsAggregate/gamesAggregate is empty methods to check stuff

Also move services functionality to these new data models